src/components/Home/Dashboard/Header/Menu/container.js

```diff
@@ -1,11 +1,29 @@
+import React from 'react';
+import PropTypes from 'prop-types';
 import { connect } from 'react-redux';
 import { compose } from 'redux';
 import { withRouter } from 'react-router-dom';
 import UserMenu from './component';
 import { authLogout } from '../../../../../store/authentifiction/actions';
 import { getUserLogin } from '../../../../../store/authentifiction/selectors';
 
-export const mstp = state => ({
+// eslint-disable-next-line no-shadow
+const MenuContainer = ({ authLogout, history, username }) => {
+  const handleLogout = () => {
+    authLogout();
+    history.push('/');
+  };
+
+  return <UserMenu username={username} handleLogout={handleLogout} />;
+};
+
+MenuContainer.propTypes = {
+  authLogout: PropTypes.func.isRequired,
+  history: PropTypes.shape(PropTypes.object).isRequired,
```

Comment:
Don't use `PropTypes.object` without description of properties of this object. Just describe it like:
```
history: PropTypes.shape({
  push: PropTypes.func.isRequired
}).isRequired
```